src/sort.rs

```diff
@@ -297,6 +308,14 @@ mod test {
         // println!("{}", sorted.to_string());
     }
 
+    #[test]
+    fn toml_workspace_deps_edit_check() {
+        let input = fs::read_to_string("examp/workspace_deps.toml").unwrap();
+        let sorted = super::sort_toml(&input, MATCHER, false, &[]);
+        assert_ne!(input, sorted.to_string());
+        // println!("{}", sorted.to_string());
```

Comment:
```suggestion
        assert_ne!(input, sorted.to_string(), "Sorted TOML: {sorted}");
```

Reply:
I don't know about the other tests in here but I think it would be better assertion if we could read a `examp/workspace_deps.sorted.toml` file that is in fact sorted.

If the code here is buggy and just adds a whitespace somewhere, this test will still pass.

Reply:
I can't push to this PR but I can send a follow-up if you'd like?